src/components/table.js

```diff
@@ -741,36 +741,56 @@ export const HeaderCell = props => {
   return h(TextCell, _.merge({ style: { fontWeight: 600 } }, props))
 }
 
+const getSortIcon = (sort, field, sortHover, style) => {
```

Comment:
The current implementation seems overly complicated. There's a lot of duplication in the two if conditions, when the only thing that varies is the icon name.